projects/aca-content/src/lib/components/details/details.component.spec.ts

```diff
@@ -140,4 +181,36 @@ describe('DetailsComponent', () => {
     fixture.detectChanges();
     expect(store.dispatch).toHaveBeenCalledWith(new SetSelectedNodesAction([node]));
   });
+
+  it('should set aspectActions from extensions', () => {
+    extensionsServiceMock.getAllowedSidebarActions.and.returnValue(of(mockAspectActions));
+    fixture.detectChanges();
+    fixture.whenStable().then(() => {
+      expect(component.aspectActions).toEqual(mockAspectActions);
+    });
+  });
+
+  it('should return the icon when getNodeIcon is called', () => {
+    const expectedIcon = 'assets/images/ft_ic_folder';
+    spyOn(contentApiService, 'getNodeIcon').and.returnValue(expectedIcon);
+    fixture.detectChanges();
+    const result = component.getNodeIcon(mockNode);
+    expect(result).toContain(expectedIcon);
+  });
+
+  it('should return the icon when getNodeIcon is called', () => {
+    const expectedIcon = 'assets/images/ft_ic_folder';
+    spyOn(contentApiService, 'getNodeIcon').and.returnValue(expectedIcon);
+    fixture.detectChanges();
+    const result = component.getNodeIcon(mockNode);
+    expect(result).toContain(expectedIcon);
+  });
+
+  it('should unsubscribe from observables on component destroy', () => {
```

Comment:
Should we test this kind of things? cc @DenysVuika

Reply:
Removed.